Parkinson/Resources/HTMLContent/15privacy_risk.html

```diff
@@ -1,11 +1,11 @@
 <body>
 
 
-<p>We take great care to protect your information, however there is a slight risk of loss of privacy.  This is a low risk because we separate your personal information (information that can directly identify you, such as your name or phone number) from the research study data to respect your privacy. Only key personnel from the study team may have the key to associate your coded study data to your name and account information if needed.<br /><br />
+<p>We take great care to protect your information, however there is a slight risk of loss of privacy.  This is a low risk because we separate your personal information (information that can directly identify you, such as your name or phone number) from the research study data to respect your privacy. Only key personnel from the study team may connect your coded study data to your name and account information if needed.<br /><br />
```

Comment:
ok this comma splice has been bugging me for a while now. can you edit it to say "...your information; however, there..." please